templates/zerver/left-sidebar.html

```diff
@@ -9,8 +9,9 @@
                           <li data-name="mentioned" class="global-filter"><span class="filter-icon"><i class="icon-vector-tag"></i></span><a href="#narrow/is/mentioned">{{ _('@-mentions') }}<span class="count"><span class="value"></span></span></a></li>
                       </ul>
                       <div id="streams_list" class="zoom-out">
-                        <div id="streams_header" class="zoom-in-hide"><h4 class="sidebar-title">{{ _('STREAMS') }}</h4>
+                        <div id="streams_header" class="zoom-in-hide"><h4 class="sidebar-title" data-toggle="tooltip" title="Filter streams list"><a href="">{{ _('STREAMS') }}</a></h4>
```

Comment:
This header is for all the streams, user is subscribed to. So, I guess unless user searches for any particular stream. title of this should not be "Filter streams list". Should be replaced by keyword like "Subscribed streams".